core/templates/dev/head/galleries/gallery.html

```diff
@@ -56,7 +56,7 @@
     <div dropdown class="dropdown navbar-left oppia-navbar-button-container oppia-gallery-category-selector protractor-test-gallery-category-selector">
       <button dropdown-toggle type="button" class="btn dropdown-toggle oppia-splash-search-input ng-cloak"
               title="<[selectionDetails.categories.description]>" style="max-width: 130px;">
-        <[selectionDetails.categories.summary | truncate:14 ]>
+        <span><[selectionDetails.categories.summary | translate:translationData.categories | truncate:14 ]></span>
```

Comment:
translationData.categories is a dictionary {'totalCount': some number}. The number of selected categories is non translatable (at least not for now, we are going to have a problem translating to Japanese or Chinese in the future). We pass this number to the translation as an argument (in this case as a filter), and it is replaced in the translated string "<[totalCount]> Categories".  Is not recommended to construct the string as a concatenation directly in the html, because different languages can place this non translatable parts in different places in the sentence. Here there is some more documentation: http://angular-translate.github.io/docs/#/guide/06_variable-replacement  but let me know if it's not clear enough!